app/assets/stylesheets/layouts/_main-content.scss

```diff
@@ -0,0 +1,7 @@
+.main-content {
+  position: relative;
+  top: 0px;
+  left: 25%;
+  width: 50%;
+  margin-bottom: 12%;
```

Comment:
Is there anything that appears underneath the `.main-content` to push down with this?